build.gradle

```diff
@@ -259,7 +259,7 @@ subprojects { subproject ->
         testCompile "io.rest-assured:rest-assured:${restAssuredVersion}"
         testCompile group: 'org.powermock', name: 'powermock-api-mockito2', version: powermockVersion
         testCompile group: 'org.powermock', name: 'powermock-module-junit4', version: powermockVersion
-        testCompile group: 'uk.gov.hmcts', name: 'befta-fw', version: '6.10.0'
+        testCompile group: 'uk.gov.hmcts', name: 'befta-fw', version: '7.0.0-RDM-10539-1'
```

Comment:
This will need updating when the following befta-fw updates are merged, see hmcts/befta-fw#149.